KsApi/models/graphql/adapters/FetchProjectsEnvelope+FetchBackerProjectsQueryData.swift

```diff
@@ -14,10 +14,33 @@ extension FetchProjectsEnvelope {
     }
 
     let envelope = FetchProjectsEnvelope(
+      type: .backed,
       projects: projects,
-      cursor: data.projects?.pageInfo.startCursor,
-      hasPreviousPage: data.projects!.pageInfo.hasPreviousPage,
-      totalCount: data.projects!.totalCount
+      cursor: data.me?.backedProjects?.pageInfo.endCursor,
+      hasNextPage: data.me?.backedProjects?.pageInfo.hasNextPage ?? false,
+      totalCount: data.me?.backingsCount ?? 0
+    )
+
+    return SignalProducer(value: envelope)
+  }
+
+  static func fetchProjectsEnvelope(from data: GraphAPI.FetchMySavedProjectsQuery.Data)
```

Comment:
Basically the same as above but shaped slightly differently for the different response.